File: Java/level-700.java
public class Main {
    public static void main(String[] args) {
        final int FIRST_NUMBER = 1;
        final int LAST_NUMBER = 100;

        for (int i = FIRST_NUMBER; i <= LAST_NUMBER; i++) {
            System.out.println(toFizzBuzzNumber(i));
        }
    }

    private static String toFizzBuzzNumber(int number) {
        return toFizzBuzz(number).isEmpty() ? String.valueOf(number) : toFizzBuzz(number);
    }

    private static String toFizzBuzz(int number) {
        return toFizz(number) + toBuzz(number);
    }

    private static String toFizz(int number) {
        return isFizz(number) ? "Fizz" : "";
    }

    private static String toBuzz(int number) {
        return isBuzz(number) ? "Buzz" : "";
    }

    private static boolean isFizz(int number) {
        return number % 3 == 0;
    }

    private static boolean isBuzz(int number) {
        return number % 5 == 0;
    }
}

File: PHP/level-700.php
<?php
    const FIRST_NUMBER = 1;
    const LAST_NUMBER = 100;
    for($i = FIRST_NUMBER; $i <= LAST_NUMBER; $i++) {
        echo to_fizz_buzz_number($i), "\n";
    }
    function to_fizz_buzz_number($number) {
        return empty(to_fizz_buzz($number)) ? (string) $number : to_fizz_buzz($number);
    }

    function to_fizz_buzz($number) {
        return to_fizz($number) . to_buzz($number);
    }

    function to_fizz($number) {
        return is_fizz($number) ? 'Fizz' : '';
    }

    function to_buzz($number) {
        return is_buzz($number) ? 'Buzz' : '';
    }

    function is_fizz($number) {
        return $number % 3 == 0;
    }

    function is_buzz($number) {
        return $number % 5 == 0;
    }
?>
